pkg/chains/formats/slsa/internal/external_parameters/external_parameters.go

```diff
@@ -35,3 +36,25 @@ func BuildConfigSource(provenance *v1.Provenance) map[string]string {
 	}
 	return buildConfigSource
 }
+
+// PipelineRun adds the pipeline run spec and provenance if available
+func PipelineRun(pro *objects.PipelineRunObjectV1) map[string]any {
+	externalParams := make(map[string]any)
+
+	if provenance := pro.GetRemoteProvenance(); provenance != nil {
+		externalParams["buildConfigSource"] = BuildConfigSource(provenance)
+	}
```

Comment:
I think this block needs coverage.

Reply:
Done! I added a test for remote resolvers with pipelineruns.